res/text/meson.build

```diff
@@ -742,6 +742,511 @@ gmm_to_bin_gen = generator(msgenc_exe,
     output: '@BASENAME@'
 )
 
+# Process mons in national dex order
+pokemon_subdirs = [
```

Comment:
I don't like duplicating a list like this, especially such a long one. If possible, pokemon_subdirs may want to be a file.

Reply:
Is it maybe preferable to generate these from the `pokemon` directory and then output the files into `text`? I have an idea of how I would want this to work in a refactored build-system.

Reply:
Moving these messages to the ```pokemon``` directory would certainly make generating them easier. What was your envisioned implementation?